src/ess/reflectometry/normalization.py

```diff
@@ -20,6 +20,19 @@
 )
 
 
+def reduce_to_q(da, qbins):
+    if da.bins:
+        return da.bins.concat().bin(Q=qbins)
```

Comment:
Is the `bins.concat()` necessary? Might be better to use the newer `dim` arg to `bin` instead.